inventory/byo/hosts.aep.example

```diff
@@ -0,0 +1,175 @@
+# This is an example of a bring your own (byo) host inventory
+
+# Create an OSEv3 group that contains the masters and nodes groups
+[OSEv3:children]
+masters
+nodes
+etcd
+lb
+
+# Set variables common for all OSEv3 hosts
+[OSEv3:vars]
+# SSH user, this user should allow ssh based auth without requiring a
+# password. If using ssh key based auth, then the key should be managed by an
+# ssh agent.
+ansible_ssh_user=root
+
+# If ansible_ssh_user is not root, ansible_sudo must be set to true and the
+# user must be configured for passwordless sudo
+#ansible_sudo=true
+
+# deployment type valid values are origin, online, atomic-enterprise, and openshift-enterprise
+deployment_type=atomic-enterprise
+
+# Enable cluster metrics
+#use_cluster_metrics=true
+
+# Add additional, insecure, and blocked registries to global docker configuration
+# For enterprise deployment types we ensure that registry.access.redhat.com is
+# included if you do not include it
+#cli_docker_additional_registries=registry.example.com
+#cli_docker_insecure_registries=registry.example.com
+#cli_docker_blocked_registries=registry.hacker.com
+
+# Alternate image format string. If you're not modifying the format string and
+# only need to inject your own registry you may want to consider
+# cli_docker_additional_registries instead
+#oreg_url=example.com/openshift3/ose-${component}:${version}
```

Comment:
Ok, should be good to go now.

On Wed, Nov 25, 2015 at 4:44 PM, Jason DeTiberus <notifications@github.com>
wrote:

> In inventory/byo/hosts.aep.example
> <https://github.com/openshift/openshift-ansible/pull/946#discussion_r45922594>
> :
>
> > +deployment_type=atomic-enterprise
> > +
> > +# Enable cluster metrics
> > +#use_cluster_metrics=true
> > +
> > +# Add additional, insecure, and blocked registries to global docker configuration
> > +# For enterprise deployment types we ensure that registry.access.redhat.com is
> > +# included if you do not include it
> > +#cli_docker_additional_registries=registry.example.com
> > +#cli_docker_insecure_registries=registry.example.com
> > +#cli_docker_blocked_registries=registry.hacker.com
> > +
> > +# Alternate image format string. If you're not modifying the format string and
> > +# only need to inject your own registry you may want to consider
> > +# cli_docker_additional_registries instead
> > +#oreg_url=example.com/openshift3/ose-${component}:${version} <http://example.com/openshift3/ose-$%7Bcomponent%7D:$%7Bversion%7D>
>
> We should probably show aep names in oreg_url here
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/openshift-ansible/pull/946/files#r45922594>.
>